recipes/ls-gkm/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: ls-gkm
+  version: 164a4a4
```

Comment:
These version numbers will cause a lot of trouble later on if there will be a release. As this version will be higher I guess.
Can you simple call the release 0.0.1 or similar?